tasks/php.js

```diff
@@ -72,6 +73,12 @@ module.exports = function (grunt) {
 				args.push('-c', options.ini);
 			}
 
+			if (options.settings) {
+				for (var key in options.settings) {
+					args.splice(0, 0, '-d', key + '=' + options.settings[key]);
+				}
+			}
```

Comment:
If I see it correctly, `dargs` prefixes every argument with `--`, but the CLI wants, that every directive starts with a `-d`, e.g. `-d date.timezone=Europe/Berlin -d log_errors=1 -d error_log=/var/log/php.log`